### ISSUE TITLE
phget doesn't return

### ISSUE THREAD

**Issue by EdmundLeex** [monkeysuffrage/phantom_mechanize#1]
phget seems doesn't return, tried with a few different urls.

**Comment by monkeysuffrage** [comment]
Which phantomjs do you have? Is it in your path?

**Comment by EdmundLeex** [comment]
I would assume so, since I use rbenv. And I definitely required phantomjs. I am using 1.9.8.0.

**Comment by monkeysuffrage** [comment]
I don't know what rbenv has to do with anything. If you type phantomjs and it runs, then it's in your path.

**Comment by EdmundLeex** [comment]
Oh, I see what you mean. Yes I do have it in my path.

**Comment by monkeysuffrage** [comment]
try doing this in irb:
require 'phantom_mechanize'
agent = Mechanize.new
page = agent.phget 'http://www.google.com'

**Comment by monkeysuffrage** [comment]
Actually I remember that there's an issue with cookies, they get too long to send to phantomjs on the command line so I recommend doing agent.cookie_jar.clear!  between requests until I figure out a better way.

**Comment by EdmundLeex** [comment]
Trying it again. Now I get something different other than going in a possible infinite loop. Sending you what I got.
web_scrapper (master) $ pry
[1] pry(main)> require 'phantom_mechanize'
agent = Mechanize.new
page = agent.phget 'http://www.google.com'

=> true
[2] pry(main)> agent = Mechanize.new
=> #<Mechanize
 #<Mechanize::CookieJar:0x007fd884370f28 @store=#<HTTP::CookieJar::HashStore:0x007fd88443af08 @gc_index=0, @gc_threshold=150, @jar={}, @logger=nil, @mon_count=0, @mon_mutex=#<Mutex:0x007fd88443ba48>, @mon_owner=nil>>
 nil>
[3] pry(main)> page = agent.phget 'http://www.google.com'

JSON::ParserError: 757: unexpected token at 'TypeError: 'undefined' is not a function (evaluating 'networkRequest.setHeader('Host')')

  /Users/user/.rbenv/versions/2.2.1/gemsets/project-gemset/gems/phantom_mechanize-0.0.4/js/phget.js:71
TypeError: 'undefined' is not a function (evaluating 'networkRequest.setHeader('Host')')

  /Users/user/.rbenv/versions/2.2.1/gemsets/project-gemset/gems/phantom_mechanize-0.0.4/js/phget.js:71
[
  {
    "domain": ".google.com",
    "expires": "Fri, 10 Jun 2016 04:12:02 GMT",
    "expiry": 1465531922,
    "httponly": true,
    "name": "NID",
    "path": "/",
    "secure": false,
    "value": "74=RrAsH_KkqPj7k-5ElN3-8Y39tvLCZ4aQZzRjJfEI9iU38mnWlbIEvP4tRkvpMKNxOdymSI56Iv1CSATgv9Te0hMFi47LyeNb44UGnK7elrNK3_oU7EmGFi-FSQx1QiTM-Z457HJ8peBXVN8"
  },
  {
    "domain": ".google.com",
    "expires": "Thu, 31 Dec 2015 16:02:17 GMT",
    "expiry": 1451577737,
    "httponly": false,
    "name": "PREF",
    "path": "/",
    "secure": false,
    "value": "ID=1111111111111111:FF=0:TM=1449720722:LM=1449720722:V=1:S=29UuwOB29vdqzKPD"
  }
]'
from /Users/user/.rbenv/versions/2.2.1/gemsets/project-gemset/gems/json-1.8.3/lib/json/common.rb:155:in `parse'
[4] pry(main)>

**Comment by monkeysuffrage** [comment]
I think that might be caused by not having the latest phantomjs, I tested with 2.0.0, I will try to address that one for next time.

**Comment by EdmundLeex** [comment]
hmmm. Let me tried that. Thanks! You might want to specify the current dependency on readme. :)

**Comment by monkeysuffrage** [comment]
If you want to make that change and do a pull request I'll accept it.

**Comment by EdmundLeex** [comment]
Perfect! Just upgraded to phantomjs2 and it works. I’ll make that pull request. And I might look into your gem when I get less busy. It looks very interesting!
Thanks again!

**Comment by EdmundLeex** [comment]
Hey there, did you get to try this on a more complex page? It was working pretty like a champ when I tested it with some major sites.
I am trying to scrape this site that uses Angular JS. It seems it’s stuck somewhere. It’s been over 20min, but it’s not returning yet. Any idea? Appreciate it!

**Comment by monkeysuffrage** [comment]
Yes, I need to work on it when I find the time, I'm going to be making fixes / changes as I need them but anyone who feels like contributing is welcome to do so.

**Comment by EdmundLeex** [comment]
Just realized I had a bad internet connection and didn’t make the pull request. Making it as promised…
Happy holiday!

**Comment by monkeysuffrage** [comment]
Thanks Edmund!
